### PR TITLE
refactor!: Removal of manual code to add provision watchers

### LINKED ISSUE #123: Remove code that loads static provision watchers (functionality now in latest SDK)
# 🚀 Tech Debt

### Relevant Package [**REQUIRED**]

 This feature request is for Driver 

### Description [**REQUIRED**]
 SDK now loads static provision watchers like it already did for devices and profiles. Service no longer needs to do this.

### Describe the solution you'd like
 Remove  addProvisionWatchers() code 
https://github.com/edgexfoundry/device-usb-camera/blob/main/internal/driver/driver.go#L401-L402

Note the folder name the SDK uses is `provisionwatchers`, so the `provision_watchers` folder must be renamed.

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

### PR DESCRIPTION
**Pull request by presatish**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: internal/driver/constants.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ const (
 	RtspTcpPort                = "RtspTcpPort"
 	DefaultRtspTcpPort         = "8554"
 	ProvisionWatcherDir        = "ProvisionWatcherDir"
-	DefaultProvisionWatcherDir = "res/provision_watchers"
+	DefaultProvisionWatcherDir = "res/provisionwatchers"
```
Comment (lenny-goodell) [inline internal/driver/constants.go]: Why is this still needed?

File: internal/driver/constants.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ const (
 	RtspTcpPort                = "RtspTcpPort"
 	DefaultRtspTcpPort         = "8554"
 	ProvisionWatcherDir        = "ProvisionWatcherDir"
```
Comment (lenny-goodell) [inline internal/driver/constants.go]: No longer needed?

**Review by lenny-goodell — DISMISSED** [review]
LGTM

**Review by lenny-goodell — DISMISSED** [review]

**Comment by cloudxxx8** [comment]
@presatish please resolve the conflict

**Review by lenny-goodell — APPROVED** [review]

**Pull request merged**